packages/jsdoc-plugins/lib/doclet.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
The extension of this file should be `jsdoc`. See: https://github.com/ckeditor/ckeditor5-engine/blob/75397b2e54f2ef3e4fe153b6dae0c72417d3ef5f/src/dataprocessor/dataprocessor.jsdoc.